contrib/gmic/template.py

```diff
@@ -33,19 +30,31 @@
 license = "CECILL-2.1"
 url = "https://gmic.eu"
 source = f"https://gmic.eu/files/source/gmic_{pkgver}.tar.gz"
-sha256 = "55993e55a30fe2da32f9533b9db2a3250affa2b32003b0c49c36eec2b2c6e007"
+sha256 = "9c00a2b4c33ff4879860eadb89f6c836062433a5debdaffbf41f98bfa83081f3"
 # vis broken
 # FIXME int: gmic_qt PreviewWidget::updateOriginalImagePosition
 hardening = ["!int"]
 # no tests
-# breaks in parallel if lib isn't built first by itself
-options = ["!check", "!parallel"]
+options = ["!check"]
+
+tool_flags = {
```

Comment:
have you actually checked this 

because unlike in gcc, -O2 and -O3 are practically the same in clang

Reply:
tbh mostly just what i remembered from some old gcc tests. goood point and now that i measure it pretty much all the filters are identical in some quick benches.

i also remember that e.g. with gcc13 flac -8 was 30% faster or so with -O3 over -O2, but here it's identical with both (just checked). good to keep in mind :)